plugins/cost-insights/src/types/Entity.ts

```diff
@@ -31,8 +30,15 @@ export interface Entity {
   An entity could be atomic or composite. An atomic entity is indivisible
   and cannot be broken into sub-entities.
 
-  A composite entity can be broken down recursively into sub-entities
-  that generate cost **over the same time period**. All costs must sum to the root cost.
+  A composite entity is divided into sub-entities that account for portions
+  of the total cost **over the same time period**. The root entity is
+  expected to only have _one_ Record consisting of the sub-entities to display
+  in the product panel (keyed by the entity type, such as "service" for
```

Comment:
Missing closing parens

Reply:
I composed a commit-message haiku for this amazing spot ❤️